### PR TITLE
Release v1.4.1

### PR DESCRIPTION
**Pull request by rhmdnd**

### CODE REVIEW

File: bundle/manifests/compliance-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1653,4 +1653,5 @@ spec:
     name: operator
   - image: ghcr.io/complianceascode/k8scontent:latest
     name: profile
-  version: 1.4.0
+  replaces: compliance-operator.v1.4.0
```
Comment (rhmdnd) [inline bundle/manifests/compliance-operator.clusterserviceversion.yaml:1656]: The NVR part of this change needs to be incorporated into `Makefile` tooling.

**Comment by rhmdnd** [comment]
Failed to provision a ROSA environment. However, we're not supporting ROSA, yet.

**Review by yuumasato — APPROVED** [review]
/lgtm

The vulnerability pointed out by ` trivy-container-scan / Security Scan for OpenSCAP image` will be addressed soon in the next release when UBI image gets updated.

**Comment by yuumasato** [comment]
Added labels, CO v.1.4.1 is out the door downstream as well:
https://access.redhat.com/errata/RHBA-2024:1830

File: bundle/manifests/compliance-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (yuumasato) [reply bundle/manifests/compliance-operator.clusterserviceversion.yaml:1656]: @rhmdnd https://github.com/ComplianceAsCode/compliance-operator/pull/524

**Pull request merged**